python/pdo/test/contract.py

```diff
@@ -483,6 +510,8 @@ def Main() :
     parser.add_argument('--eservice-db', help='json file for eservice database', type=str)
     parser.add_argument('--eservice-name', help='List of enclave services to use. Give names as in database', nargs='+')
     parser.add_argument('--eservice-url', help='List of enclave service URLs to use', nargs='+')
+    parser.add_argument('--randomize-eservice', help="Say True if eservice must be randomized for each update. \
+        Else, the same eservice (the first one in the list of input eservices) will be used for all udpates.", type=bool, default=False)
```

Comment:
this is not the way to get a boolean parameter. should be
action=store_true

Reply:
thank you, will fix this